provider/rancher/rancher.go

```diff
@@ -462,6 +469,36 @@ func (lbp *LBProvider) getLBServiceForConfig(lbConfigName string) (*client.LoadB
 	return lbp.getLBServiceByName(fmtName)
 }
 
+func schedulerLabelsChanged(oldLabels map[string]interface{}, newLabels map[string]string) bool {
+	for k, v1 := range newLabels {
+		if !strings.HasPrefix(k, rancherSchedulerPrefix) {
+			continue
+		}
+		key := rancherSchedulerPrefix + strings.Replace(k[len(rancherSchedulerPrefix):], ".", ":", 1)
+		v2, ok := oldLabels[key]
+		if !ok {
+			return true
+		}
+		if v1 != v2.(string) {
```

Comment:
@wlan0 can you double check if the labels' values are case sensitive in Rancher? If not, do the comparison ignoring the case